mmpose/core/evaluation/top_down_eval.py

```diff
@@ -202,6 +202,30 @@ def keypoint_auc(pred, gt, mask, normalize, num_step=20):
     return auc
 
 
+def keypoint_nme(pred, gt, mask, normalize):
+    """Calculate the end-point error.
```

Comment:
what does `nme` mean

Reply:
`normalized mean error (NME)` I will add this information.